jdi-light-material-ui-tests/src/test/java/io/github/epam/material/tests/displaydata/ChipTests.java

```diff
@@ -13,7 +13,7 @@ public void defaultChipTest(){
         displayDataDefaultChipPage.open();
 
         basicCheck();
-        disabledCheck(); // first variant of class value
+        disabledCheck();
```

Comment:
Please return comment, as it's information that could help

Reply:
done